data/hai missions.txt

```diff
@@ -1061,3 +1061,251 @@ mission "Hai Honeymoon"
+			`	As you're trying to figure out a way to get the most out of the available missions, you feel a tap on your shoulder, and turn to see a Hai standing beside a human couple. The couple introduce themselves as Xavier and Vanessa, who are here with their Hai friend Narancia. "We overheard your conversation and are willing to pay a little extra for a job. We're part of a group organizing a big festival in Allhome to celebrate the friendship between humans and Hai, and we just got a last-minute idea: we'd like to get our hands on some Quarg spices."`
+			`	They tell Nanachi that they won't be going along, they already have a transport to Allhome, and will stay here a bit longer to try and advertise the event.`
+				accept
+	on stopover
+		dialog
+			`A spaceport worker contacted Nanachi shortly before you two landed and made sure the spices were ready to be loaded in her ship. You help the workers and Nanachi make sure everything is in order, and prepare for your trip to Allhome.`
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog `You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system.`
+	on complete
+		conversation
+			`Allhome's docking area shows much more traffic than usual. Many captains here are helping workers open cargo containers full of food, clothes, books, movies and even some mounted animals from human space. You finally meet up with the festival's organizers, who seem very excited to have acquired the exotic condiments, and have invited you two to a bar in the spaceport for a drink.`
+
+
+
+mission "Nanachi 4"
+	name "Protect Nanachi"
+	description "Bring Nanachi's ship to Prime, keeping her safe from a pirate fleet that attacked a convoy heading to Hai space, then head back to <planet>."
+	source "Allhome"
+	stopover "Prime"
+	deadline 14
+	to offer
+		has "Nanachi 3: done"
+	on offer
+		conversation
+			`The bar is just as busy as the docks. Many captains here must have transported something for the festival. "Captain <last>, captain Nanachi!" shouts Narancia, beckoning the two of you to join her and the others at their table. You learn that they were inspired by the Hai's own March festival, and wanted to bring small parts of human culture to Hai space.`
+			`	"They can't go to our planets and experience it for themselves yet, so we thought about bringing it to them!" Vanessa says.`
+			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the European renaissance!" Xavier adds.`
```

Comment:
```suggestion
			`	"It was very hard to get enough sponsors for this, but after years of hard work we even managed to have famous art pieces be brought here, some of which date back to the European Renaissance!" Xavier adds.`
```
I'm curious as to what they got their hands on. 🤔 I imagine Michelangelo's David surrounded by tables of snacks.